src/main/java/functional/tests/core/mobile/device/android/AndroidDevice.java

```diff
@@ -441,7 +441,9 @@ public void startActivity(String appPackage, String appActivity) {
     private void startEmulator() throws DeviceException, TimeoutException {
 
         // Kill all simulators not matching framework convention
-        this.stopWrongPortEmulators();
+        if (!this.settings.reuseDevice) {
```

Comment:
We can run with reuseDevice=false on CI.
Can we check this.settings.debug?

Reply:
@dtopuzov Sure, I thought it will be more comprehensible if it is a separate option, but if we all agree that adding debug=true to the properties file is enough to run tests locally without closing the emulator when not debugging, I'm fine.